src/main/java/Duke.java

```diff
@@ -1,10 +1,82 @@
+import java.util.Scanner;
+
 public class Duke {
+    static String lineCutOff = "_______________________";
+    static int TODO_TASK_INDEX = 5;
+    static int DEADLINE_TASK_INDEX = 9;
+    static int EVENT_TASK_INDEX = 6;
+
+    public static Task taskType(String task) {
+        if (task.startsWith("todo")){
+            return new ToDo(task.substring(TODO_TASK_INDEX ));
+        }else if (task.startsWith("deadline")){
+            int indexBy = task.indexOf("/");
+            return new Deadline(task.substring(DEADLINE_TASK_INDEX,indexBy - 1), task.substring(indexBy + 3));
+        }else if (task.startsWith("event")){
+            int indexAt = task.indexOf("/");
+            return new Event(task.substring(EVENT_TASK_INDEX, indexAt - 1), task.substring(indexAt + 3));
+        }
```

Comment:
Should the formatting here be modified such that there is an extra space between the end of the parenthesis and the opening brackets `){` vs `) {` , as well as between the end of the bracket and the start of an `else if`?